### PR TITLE
disable boot_cpu_model test case for ppc64le platform since it's not …

### PR DESCRIPTION
**Pull request by qunfang**
ID:1263995

### CODE REVIEW

File: qemu/tests/cfg/boot_cpu_model.cfg [hunk 1]
```diff
@@ -1,4 +1,5 @@
 - boot_cpu_model:
+    only x86_64 i386
```
Comment (suqinhuang) [inline qemu/tests/cfg/boot_cpu_model.cfg]: you can disable in Linux/RHEL/7.1/ppc64le.cfg directly

**Comment by CongSmile** [comment]
Hi @qunfang 

How about arm?
I think disable ppc is fine for the case.

Thanks.

**Comment by qunfang** [comment]
@CongSmile thanks for the comment, changed it.

**Comment by CongSmile** [comment]
Acked-by: Cong Li <coli@redhat.com>

**Comment by xutian** [comment]
Merged, thanks @qunfang and @CongSmile

**Pull request merged**